### PR TITLE
Add read function for EPS sensors

### LINKED ISSUE #213: Add read function for EPS sensors
### Description

We want to read all sensors as at-the-same-time as possible and don't care about units. Everything gets forwarded as it is without conversion.

### PR DESCRIPTION
**Pull request by danielschloms**
### Description

This PR adds functionality to read the EPS sensors, including a test. What is left to do is testing this on the HW and discussing and choosing the final configuration for the averaging register.

Fixes #213

### CODE REVIEW

File: Sts1CobcSw/Periphery/Eps.cpp [hunk 1]
```diff
@@ -135,4 +156,67 @@ auto ConfigureSetupRegister(hal::GpioPin * adcCsPin) -> void
...
+
+auto ConfigureAveragingRegister(hal::GpioPin * adcCsPin) -> void
+{
+    // Averaging register values
+    // [7:5]: Register selection bits = 0b001
+    // [4]: Averaging on/off
+    // [3:2]: Number of conversions used
+    // [1:0]: Single-channel scan count (scan mode 0b10 in conversion only)
+
+    // TODO: discuss and chose averaging values
```
Comment (danielschloms) [inline Sts1CobcSw/Periphery/Eps.cpp:169]: chose -> choose

**Comment by danielschloms** [comment]
This was tested by checking the battery voltage with the COBC V2.6 ID 3, and it looked correct.

**Comment by PatrickKa** [comment]
We need to do the HW tests again because we changed `eps::Read()` and `eps::ReadAdc()`.

**Comment by PatrickKa** [comment]
I did the HW tests today and they worked.

**Pull request merged**